app/models/search_timdex.rb

```diff
@@ -0,0 +1,35 @@
+# Searches TIMDEX! API and formats results into {Result} object
+#
+# == Required Environment Variables:
+# - TIMDEX_URL
+#
+# == Optional Environment Variables:
+# - RESULTS_PER_BOX
+#
+# == See Also
+# - https://timdex.mit.edu/playground
+class SearchTimdex
+  attr_reader :results
+
+  TIMDEX_URL = ENV['TIMDEX_URL'].freeze
+
+  def initialize
+    @timdex_http = HTTP.persistent(TIMDEX_URL)
```

Comment:
I think this persistent connection setup here isn't used afterwards? We should use it though so moving the headers config up here and then calling the post using this setup object would be 🌈 

I'm imagining something like (which drops some headers I don't think we need and updates origin:
```
    @timdex_http = HTTP.persistent(TIMDEX_URL)
    @timdex_http.headers(accept: 'application/json',
      'Accept-Encoding': 'gzip, deflate, br',
      'Content-Type': 'application/json',
      'Origin': 'https://lib.mit.edu'
    )
```

Reply:
This change has been implemented. I ended up needing a single statement here rather than two, as you'll see in the diff.